client/src/app/+login/login.component.scss

```diff
@@ -1,12 +1,10 @@
 @use '_variables' as *;
 @use '_mixins' as *;
 
-@import 'bootstrap/scss/functions';
-@import 'bootstrap/scss/variables';
-
 label {
+  @include font-size(1.125rem);
```

Comment:
Please use a variable instead of `1.125rem` (like you did with `font-size-rem-14px`)